docs/devel/coding-conventions.md

```diff
@@ -30,12 +30,31 @@ Documentation for other releases can be found at
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-Coding style advice for contributors
+Code conventions
   - Bash
     - https://google-styleguide.googlecode.com/svn/trunk/shell.xml
   - Go
     - https://github.com/golang/go/wiki/CodeReviewComments
-    - https://gist.github.com/lavalamp/4bd23295a9f32706a48f
+    - Command-line flags should use dashes, not underscores
+
+Testing conventions
+  - All new packages and most new significant functionality must come with unit tests
+  - Table-driven tests are preferred for testing multiple scenarios/inputs; for example, see [TestNamespaceAuthorization](../../test/integration/auth_test.go)
```

Comment:
I don't necessarily think that prescribing one is a good idea, but it would be good to try to stick to an existing convention that may exist in any given file.